config.py

```diff
@@ -89,6 +89,22 @@
 The runs to process.
 """
 
+exclude_runs: Optional[Dict[str, List[str]]] = None
```

Comment:
I think we should support both, a dict of lists (as you do now) **and** a list of strings (as we do with `exclude_subjects`) to exclude the same run across all subjects. WDYT?

Reply:
cc @agramfort (see my previous comment)

Reply:
I thought about it, but I think that the usage of this exclude_runs should be parsimonious. And if I want to exclude a run for all subjects I can just delete it from the list of runs in the runs list.

Reply:
I think we can add this later if requested

Reply:
Ok fair enough